tests/tests/test_monitor_client.py

```diff
@@ -657,3 +663,73 @@ def test_dbus_subsystem(self, monitor_commercial_setup_no_client):
             + devid
         )
         logger.info("test_dbus_subsystem: got CRITICAL alert email.")
+
+    def test_dbus_pattern_match(self, monitor_commercial_setup_no_client):
+        """Test the dbus subsystem"""
+        mailbox_path = "/var/spool/mail/local"
+        wait_for_alert_interval_s = 8
+        expected_from = "alert@mender.io"
+        dbus_name = "test"
+        user_name = "bugs.bunny@acme.org"
+        devid, _, _, mender_device = self.prepare_env(
+            monitor_commercial_setup_no_client, user_name
+        )
+        logger.info("test_dbus_subsystem: env ready.")
+
+        logger.info(
+            "test_dbus_subsystem: email alert on dbus signal pattern match scenario."
+        )
+        prepare_dbus_monitoring(mender_device, dbus_name, log_pattern="mender")
+        time.sleep(2 * wait_for_alert_interval_s)
+
+        mail = monitor_commercial_setup_no_client.get_file("local-smtp", mailbox_path)
+        messages = parse_email(mail)
+
+        assert len(messages) > 0
+        m = messages[0]
+        assert "To" in m
+        assert "From" in m
+        assert "Subject" in m
+        assert m["To"] == user_name
+        assert m["From"] == expected_from
+        assert (
+            m["Subject"]
+            == "CRITICAL: Monitor Alert for D-Bus signal arrived on bus system bus on "
+            + devid
+        )
+        logger.info("test_dbus_subsystem: got CRITICAL alert email.")
+
+    def test_dbus_bus_filter(self, monitor_commercial_setup_no_client):
+        """Test the dbus subsystem"""
+        mailbox_path = "/var/spool/mail/local"
+        wait_for_alert_interval_s = 8
+        expected_from = "alert@mender.io"
+        dbus_name = "test"
+        user_name = "bugs.bunny@acme.org"
+        devid, _, _, mender_device = self.prepare_env(
+            monitor_commercial_setup_no_client, user_name
+        )
+        logger.info("test_dbus_subsystem: env ready.")
+
+        logger.info(
+            "test_dbus_subsystem: email alert on single dbus filter signal scenario."
+        )
+        prepare_dbus_monitoring(mender_device, dbus_name, log_pattern="mender")
```

Comment:
in both places you are passing log_pattern, so, where are we testing WATCH_PATTERN